src/Forms/CompositeField.php

```diff
@@ -214,7 +214,8 @@ public function getAttributes()
                 'tabindex' => null,
                 'type' => null,
                 'value' => null,
-                'title' => ($this->tag == 'fieldset') ? null : $this->legend
+                'title' => ($this->tag == 'fieldset') ? null : $this->legend,
```

Comment:
There's no unit test for this method right now. If you feel like adding some, it would be cool. But I'm not overly fuss about it.